web/cobrands/sass/_base.scss

```diff
@@ -2975,9 +2980,15 @@ a#geolocate_link {
   }
 }
 
+.cta-box__input-wrapper {
+  display: flex;
+  gap: 0.5rem;
+  flex-wrap: wrap;
```

Comment:
This means it's always two lines at mobile width, is that what's wanted? Seems to be okay on the one line at all widths to me.

Reply:
I'm not too bother by having only one line, but the advantage of having two lines is for users with longer emails can see their whole input.

Reply:
Yeah, it's fine at actual mobile, I guess it's only at 'nearly desktop' width it looks a little odd, probably not worth doing anything about.